elide-datastore/elide-datastore-aggregation/src/main/java/com/yahoo/elide/datastores/aggregation/metadata/MetaDataStore.java

```diff
@@ -35,22 +37,47 @@
 import java.util.List;
 import java.util.Map;
 import java.util.Set;
+import java.util.function.Function;
 import java.util.stream.Collectors;
 
 /**
  * MetaDataStore is a in-memory data store that manage data models for an {@link AggregationDataStore}.
  */
-public class MetaDataStore extends HashMapDataStore {
+public class MetaDataStore implements DataStore {
     private static final Package META_DATA_PACKAGE = Table.class.getPackage();
 
     private static final List<Class<? extends Annotation>> METADATA_STORE_ANNOTATIONS =
             Arrays.asList(FromTable.class, FromSubquery.class, Subselect.class, javax.persistence.Table.class);
 
+    public static final Function<String, HashMapDataStore> ERROR_OUT = new Function<String, HashMapDataStore>() {
+        @Override
+        public HashMapDataStore apply(String key) {
+            throw new IllegalStateException("API version " + key + " not found");
```

Comment:
HttpStatusException throw some 4XX error.

Reply:
changed to BadRequestException